packages/selinux-policy/fs.cil

```diff
@@ -51,6 +51,8 @@
 (filecon "/.*/usr/sbin/chronyd" file clock_exec)
 (filecon "/.*/usr/sbin/wicked.*" file network_exec)
 (filecon "/.*/usr/libexec/wicked/bin/wicked.*" file network_exec)
+(filecon "/.*/usr/lib/systemd/systemd-networkd" file network_exec)
```

Comment:
I wonder if you could do what is already in the policy for wicked a few lines above:

```
(filecon "/.*/usr/lib/systemd/systemd-networkd*" file network_exec)
```